TestApp/update-npm-packages.sh

```diff
@@ -13,32 +42,50 @@ npm pack ../appcenter-link-scripts
 echo "Installing appcenter* packages..."
 npm install appcenter*.tgz
 
-echo "workaround for macs on arm64"
-npm install react-native-fs --save --legacy-peer-deps
-
 echo "Cleanup appcenter*.tgz"
 rm appcenter*.tgz
 
 echo "Installing other packages..."
 npm install
 
-# Issue: https://github.com/react-native-community/cli/issues/1403
-# Fixed in RN 0.65: https://github.com/facebook/react-native/commit/8721ee0a6b10e5bc8a5a95809aaa7b25dd5a6043
-echo "Applying patch to react-native package that fixes build on Xcode 12.5..."
-npx patch-package
+if [ "$is_ios" == "true" ]; then
+    echo "Workaround for macs on arm64"
+    npm install react-native-fs --save --legacy-peer-deps
```

Comment:
This workaround can removed since on macos12 image lowest version of [xcode](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#xcode) is 13.1 so this workaround is no longer viable.